### ISSUE TITLE
Rotate image at tagging time phase

### ISSUE THREAD

**Issue by kichinsky** [Microsoft/VoTT#310]
Per a customer request:
-- Allow image rotation to simplify tagging (should not affect original image and export).

I think it is related to zooming #247 in the implementation.

**Comment by wbreza** [comment]
@kichinsky - are you referring to common tooling in other image related apps where the user can rotate the image in 90 degree increments?

**Comment by kichinsky** [comment]
Yes, exactly, like change orientation from vertical to horizontal. From the implementation point of view, I'm not sure what is the best approach, like temporary rotate source image or to virtually rotate it on the screen with some recalculation of coordinates.

**Comment by JCAAUSFRA** [comment]
any update on how the zooming part of this is going? Super Useful feature!

**Comment by apetkov** [comment]
Can you give us a timeline when we can expect zooming to be available? VoTT would be super useful to us, as long as we can zoom.
Thanks!
